GHzDACs/ghz_fpga_server.py

```diff
@@ -1627,16 +1627,66 @@ def dac_run_sram(self, c, data, loop=False, blockDelay=0):
         """
         if not len(data):
             return
-        
-        if loop:
-            # make sure data is at least 20 words long by repeating it
-            data *= (20-1)/len(data) + 1
-        else:
-            # Set data at least 20 words long by repeating first value
+
+        if len(data) < 20:
+            # pad out to 20 with zeros
+            d2 = np.zeros(20)
+            d2[0:len(data)] = data
+            data = d2
```

Comment:
I don't exactly remember. I was trying to get the JT to work and... I don't remember. But it was necessary at the time. Probably.